globalize-compiler/globalize-compiler.d.ts

```diff
@@ -0,0 +1,106 @@
+// Type definitions for globalize-compiler v0.2.0
+// Project: https://github.com/jquery-support/globalize-compiler
+// Definitions by: Ian Clanton-Thuon <https://github.com/iclanton>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="./../globalize/globalize.d.ts" />
+/// <reference path="../estree/estree.d.ts" />
+
+declare namespace GlobalizeCompiler {
```

Comment:
The [TypeScript Style Guide](https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines) recommends PascalCase for types.

Numerous examples in DT set precedent. Clicking around randomly:
- https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/amcharts/AmCharts.d.ts#L7
- https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/chai-dom/chai-dom.d.ts#L8
- https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/hystrixjs/hystrixjs.d.ts#L9